solutions/banking/outputs.tf

```diff
@@ -0,0 +1,44 @@
+output "project_id" {
+  description = "ID of the created project."
+  value       = module.configure_project.project_id
+}
+
+output "project_url" {
+  description = "Project ID URL."
+  value       = "https://dataplatform.cloud.ibm.com/projects/${module.configure_project.project_id}"
+}
+
+output "watsonx_assistant_url" {
+  description = "WastonX Assistant URL."
+  value       = local.watsonx_assistant_url
+}
+
+output "watsonx_discovery_url" {
+  description = "WastonX Discovery URL."
+  value       = local.watsonx_discovery_url
+}
+
+output "cos_instance_guid" {
+  description = "COS instance GUID."
```

Comment:
Indicate what COS instance this is - I suppose this is the one associated with watsonx.ai project?